lib/galaxy/managers/tool_data.py

```diff
@@ -0,0 +1,81 @@
+
+from pathlib import Path
+from typing import Dict
+
+from galaxy import exceptions
+from galaxy.app import UniverseApplication
+from galaxy.queue_worker import GalaxyQueueWorker
+from galaxy.tools.data import (
+    TabularToolDataField,
+    TabularToolDataTable,
+)
+from galaxy.tools.data._schema import (
+    ToolDataDetails,
+    ToolDataEntryList,
+    ToolDataField,
+)
+
+
+class ToolDataManager:
+    """
+    Interface/service object for interacting with tool data.
+    """
+
+    def __init__(self, app: UniverseApplication):
+        self._app = app
+
+    @property
+    def data_tables(self) -> Dict[str, TabularToolDataTable]:
+        return self._app.tool_data_tables.data_tables
+
+    @property
+    def queue_worker(self) -> GalaxyQueueWorker:
```

Comment:
If it isn't meant to be part of the public interface of the class I guess we should prefix the name with ``_``. I think this should probably be ``_queue_worker``.

Reply:
Actually if you just use ``self._app.queue_worker`` in the one place this is used you get all the typing benefit with the new StructureApp approach but you don't need to import ``GalaxyQueueWorker`` and make it explicit like this. I could go either way here though - this helper wouldn't be my preference but making this a little more explicit here might improve readability?

Reply:
Absolutely!, I missed it, thanks for the catch :)

Reply:
I like your suggestion of using `self._app.queue_worker` and remove the attribute and the import completely. It is used only once and it's quite readable and less code.